src/cli/CMakeLists.txt

```diff
@@ -283,11 +283,18 @@ if(BUILD_TESTING)
       WORKING_DIRECTORY "${PROJECT_BINARY_DIR}/resources/workflow/outdated_measures/"
     )
 
-  add_test(NAME OpenStudioCLI.test_output_files
+    add_test(NAME OpenStudioCLI.test_output_files
       COMMAND ${Python_EXECUTABLE} -m pytest --verbose --os-cli-path $<TARGET_FILE:openstudio> "${CMAKE_CURRENT_SOURCE_DIR}/test/test_output_files.py"
       WORKING_DIRECTORY "${PROJECT_BINARY_DIR}/resources/workflow/output_test/"
     )
 
+    file(MAKE_DIRECTORY "${PROJECT_BINARY_DIR}/Testing/")
+    add_test(NAME OpenStudioCLI.test_bcl_measure_templates
+      COMMAND ${Python_EXECUTABLE} -m pytest --verbose --os-cli-path $<TARGET_FILE:openstudio> "${CMAKE_CURRENT_SOURCE_DIR}/test/test_bcl_measure_templates.py"
+      WORKING_DIRECTORY "${PROJECT_BINARY_DIR}/Testing/"
+    )
```

Comment:
register the pytest as a cmakelists one.

Voluntarily not using "-n auto" (pytest-xdist) so I check for loading errors (the thing del `sys.modules['measure']` fixes).

I used system pytest (because it wasn't working with the CLI at first) but we could replace it with openstudio, or do it with both.

Mind that the ReportingMeasure tests are quite slow (there's a simulation happening)

Reply:
And maybe another one for `openstudio measure --run_tests Testing/bcl_templates` once all measures have been instantiated too.

We can figure that out later